File: src/test/java/no/nav/syfo/service/DatoServiceTest.java
package no.nav.syfo.service;

import org.junit.Test;
import org.junit.runner.RunWith;
import org.mockito.runners.MockitoJUnitRunner;

import java.time.LocalDate;

import static no.nav.syfo.util.DatoService.dagerMellom;
import static org.assertj.core.api.Assertions.assertThat;

@RunWith(MockitoJUnitRunner.class)
public class DatoServiceTest {

    @Test
    public void ekskludererHelgedager() {
        int dagerMellom = dagerMellom(LocalDate.of(2017, 2, 17), LocalDate.of(2017, 2, 20));
        assertThat(dagerMellom).isEqualTo(1);
    }
}

File: src/test/kotlin/no/nav/syfo/service/DatoServiceTest.kt
package no.nav.syfo.service

import no.nav.syfo.util.DatoService
import org.assertj.core.api.Assertions
import org.junit.Test
import org.junit.runner.RunWith
import org.mockito.junit.MockitoJUnitRunner
import java.time.LocalDate

@RunWith(MockitoJUnitRunner::class)
class DatoServiceTest {
    @Test
    fun ekskludererHelgedager() {
        val dagerMellom = DatoService.dagerMellom(
            LocalDate.of(2017, 2, 17),
            LocalDate.of(2017, 2, 20)
        )
        Assertions.assertThat(dagerMellom).isEqualTo(1)
    }
}
